models/docs.md

```diff
@@ -116,9 +116,9 @@ The daily count of newly purchased subscriptions.
 How the user got to your store listing: Google Play search, Third-party referral, Google Play explore, or Other.
 {% enddocs %}
 
-{% docs country %}
-Two-letter abbreviation of the country where the user’s Google account is registered.
-{% enddocs %}
+{% docs country %} 
+Two-letter abbreviation of the country where the user’s Google account is registered. 
+{% enddocs %} 
```

Comment:
I noticed in the source data that there are a few records that have `null` for the country. Do you know by chance what this means?

Reply:
yeah like if the user has opted out of tracking and only anonymized data is passed through, tho there may be other cases i'm not aware of. included it in the decision log here https://github.com/fivetran/dbt_google_play_source/blob/main/DECISIONLOG.md#grouping-records-with-null-dimensions-into-daily-batches

Reply:
That makes total sense. Thanks for calling this out!